src/cfml/system/services/ServerService.cfc

```diff
@@ -860,6 +871,27 @@ component accessors="true" singleton {
 		serverInfo.libDirs		= ( serverProps.libDirs		?: serverJSON.app.libDirs ?: '' ).listAppend( defaults.app.libDirs );
 
 		serverInfo.webRules = [];
+
+		//ssl force redirect
+		if(serverInfo.SSLEnable && serverInfo.SSLForceRedirect){
+			serverInfo.webRules.append(
+				"not secure() and method(GET) -> redirect('https://%{LOCAL_SERVER_NAME}%{REQUEST_URL}%{QUERY_STRING}')"
+			);
+		}
+
+		//ssl hsts
+		if(serverInfo.SSLEnable && serverInfo.HSTSEnable){
+			if(serverInfo.HSTSIncludeSubDomains){
```

Comment:
@BalbinoOrtus Can you try to do this with the ternary operator like we discussed in our call?

Reply:
sure no problem